.github/workflows/testadapters.yaml

```diff
@@ -105,7 +118,6 @@ jobs:
               if [ "$svcstatus" = "<pending>" ];then
```

Comment:
@piyushsingariya where is mesheryctl system dashboard in this?

Reply:
This service status is the information about whether an external IP has been allocated or not by minikube tunnel. I dont think mesheryctl can get us the external IP of a service

Reply:
It should be. @hexxdump, will you chime in here?

Reply:
It can get external IP and PORT if minikube tunnel is active in other terminal.